frontend/ui/uimanager.lua

```diff
@@ -460,6 +462,10 @@ function UIManager:close(widget, refreshtype, refreshregion, refreshdither)
     if requested_disable_double_tap ~= nil then
         Input.disable_double_tap = requested_disable_double_tap
     end
+    if #self._window_stack > 0 then
```

Comment:
Skip this if tap interval is set to 0?

Reply:
(Although, granted, it's not technically the end of the world, and it may make the logic murkier).

Reply:
> Skip this if tap interval is set to 0?

One might have the generic tap interval (used when turning page by tap) set to 0 - but still wanted to have a tap interval on keyboard non-0.
(But not sure what you mean, stkip the `#self._window_stack>0` test only ? And the generic tap interval is not readily accessible from there, it's in GestureDetector, so, a bit of ugly reaching it via Input.gesture_detector...)